examples/mini-compile/src/Main.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE PackageImports #-}
+{-# OPTIONS_GHC -Wno-missing-fields #-}
+
+module Main (main) where
+
+import "ghc-lib" GHC
+import "ghc-lib" Paths_ghc_lib
+import "ghc-lib" HeaderInfo
+import "ghc-lib" Module
+import "ghc-lib" Config
+import "ghc-lib" DynFlags
+import "ghc-lib" Platform
+import "ghc-lib" StringBuffer
+import "ghc-lib" Fingerprint
+
+import Outputable
+import System.Environment
+import System.Directory
+import System.IO.Extra
+import qualified Data.Map.Strict as Map
+import Data.IORef
+
+main :: IO ()
+main = do
+  args <- getArgs
+  case args of
+    [file] -> do
+      s <- readFile' file
+      flags <- mkDynFlags file s
+      dataDir <- getDataDir
+      createDirectoryIfMissing True (dataDir ++ "/../mingw")
+      cm <- runGhc (Just dataDir) $ do
+              setSessionDynFlags flags
+              compileToCoreSimplified file
+      putStrLn (showSDoc flags (ppr (cm)))
```

Comment:
Run HLint over this - at least one redundant bracket

Reply:
✔️